zonos/model.py

```diff
@@ -260,13 +268,25 @@ def generate(
         max_steps = delayed_codes.shape[2] - offset
         remaining_steps = torch.full((batch_size,), max_steps, device=device)
         progress = tqdm(total=max_steps, desc="Generating", disable=not progress_bar)
-        cfg_scale = torch.tensor(cfg_scale)
+        #cfg_scale = torch.tensor(cfg_scale)
```

Comment:
This means that the export has cfg_scale hard coded. To un-hard-code it, we need to uncomment this and then fix the single error caused by branching on cfg_scale. There might be a way to torch.cond it, I attempted to solve it on stream but it wasn't obviously cond-able so I bailed.